indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -95,5 +178,47 @@
                 });
             }
         });
+
+        $('.email-preview-btn').on('click', function(evt) {
+            evt.preventDefault();
+            var id = $(this).data('id');
+            var $previewBtn = $('#email-preview-btn-' + id);
+            if ($previewBtn.data('visible')) {
+                $previewBtn.text($T.gettext('Show preview'));
+                $('#email-preview-' + id).slideToggle();
+                $previewBtn.data('visible', false);
+            } else {
+                $previewBtn.text($T.gettext('Hide preview'));
+                $('#email-preview-' + id ).slideToggle();
+                $previewBtn.data('visible', true);
+            }
+        });
+
+        $('.ui-i-box-sortable-handle').on('mousedown', function() {
```

Comment:
Don't we have a more specific selector? This would select *any* sortable on the page if there was more than one